packages/instrumentation-langchain/src/instrumentation.ts

```
@@ -16,226 +16,90 @@ import { context } from "@opentelemetry/api";
-      moduleExports.VectorStoreRetriever.prototype,
-      "_getRelevantDocuments",
-    );
-    return moduleExports;
+  private instrumentCallbackManagerDirectly() {
+    try {
+      const callbackManagerModule = require("@langchain/core/callbacks/manager");
+      
+      if (callbackManagerModule?.CallbackManager) {
+        this.patchCallbackManager(callbackManagerModule.CallbackManager);
+      } else {
+      }
```

ESLint

[error] Empty block statement.

(no-empty)